### PR TITLE
sets maxEdge default to Infinity

### PR DESCRIPTION
**Pull request by stebogit**
Ref "`@turf/concave` improvements" #910

### CODE REVIEW

File: packages/turf-concave/index.d.ts [hunk 1]
```diff
@@ -5,8 +5,8 @@ import { Point, Feature, FeatureCollection, MultiPolygon, Polygon, Units} from '
  */
 export default function concave(
     points: FeatureCollection<Point>,
-    maxEdge: number,
     options?: {
-      units?: Units
+        maxEdge: number,
```
Comment (DenisCarriere) [inline packages/turf-concave/index.d.ts]: using the `?` means it's optional, at the moment if someone would enter options, then `maxEdge` would be required.

Fix to:
```ts
options?: {
  maxEdge?: number,
  units?: Units
}

File: packages/turf-concave/bench.js [hunk 2]
```diff
@@ -24,9 +24,8 @@ const fixtures = fs.readdirSync(directory).map(filename => {
  * pts3: 0.375ms
  */
 for (const {name, geojson} of fixtures) {
-    const {maxEdge, units} = geojson.properties || {maxEdge: 1};
     console.time(name);
-    concave(geojson, maxEdge, {units});
+    concave(geojson, geojson.properties);
```
Comment (DenisCarriere) [inline packages/turf-concave/bench.js]: 👍 I'm liking this Object `options`, you can simply give it the `geojson.properties` like you did, love it!

**Comment by DenisCarriere** [comment]
👍 Looks good!

File: packages/turf-concave/bench.js (hunk 2 above)
Comment (stebogit) [reply packages/turf-concave/bench.js]: Thanks Sir 😄

**Pull request merged**